Source/WebKit2/WebProcess/Cookies/WebCookieManager.cpp

```diff
@@ -36,7 +36,6 @@
 #include <WebCore/URL.h>
 #include <wtf/MainThread.h>
 #include <wtf/text/StringHash.h>
-#include <wtf/text/WTFString.h>
```

Comment:
Keeping the include here and using the forward declaration in the header as provided by <wtf/Forward.h> should be enough(?).